include/robot_localization/ros_filter_utilities.h

```diff
@@ -97,6 +98,72 @@ void stateToTF(const Eigen::VectorXd &state, tf2::Transform &stateTF);
 //!
 void TFtoState(const tf2::Transform &stateTF, Eigen::VectorXd &state);
 
+
+//! @brief Class to determine if it is time to publish
+class TimeToPublish
+{
+
+public:
+
+  //! @brief Default constructor
+  TimeToPublish()
+    : period_(0.02)
+    , last_(ros::WallTime::now().toSec())
+    , tolerance_(0.001)
+  {}
+
+  //! @brief Constructor
+  TimeToPublish(const double period, const double last)
```

Comment:
You can fuse this one with the previous constructor using the defaults to initialize the arguments, like `TimeToPublish(const double period = 0.02, const double last = ...)`